### PR TITLE
Merge changes file with actual one in SP5

### PR DESCRIPTION
**Pull request by varkoly**
## Problem
* The changes-file in git differs from the one in build-service.

## Solution

* Merge changes with build service.

### CODE REVIEW

File: .yardopts [hunk 1]
```diff
@@ -1,7 +0,0 @@
---no-private
```
Comment (jreidinger) [inline .yardopts:1]: what is reason to drop yardopts?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -11,11 +11,9 @@ jobs:
 
     strategy:
       fail-fast: false
-      matrix:
-        distro:  [ "tumbleweed", "leap_latest" ]
 
     container:
-      image: registry.opensuse.org/yast/head/containers_${{matrix.distro}}/yast-ruby
+      image: registry.opensuse.org/yast/sle-15/sp5/containers/yast-ruby
```
Comment (jreidinger) [inline .github/workflows/ci.yml:16]: @lslezak is this change correct? It looks like sle 15 container is missing sudo

Comment (lslezak) [reply .github/workflows/ci.yml:16]: It is OK, there are wrong (old) some Ruby gems, I'm fixing that...

File: .yardopts (hunk 1 above)
Comment (varkoly) [reply .yardopts:1]: I do not know anything about it.

Comment (varkoly) [reply .yardopts:1]: IMHO yard will be not used in this package at all.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lslezak) [reply .github/workflows/ci.yml:16]: Fixed, now CI is green. :heavy_check_mark:

File: package/yast2-sap-ha.spec [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-sap-ha
-Version:        4.5.6
+Version:        4.5.1
```
Comment (lslezak) [inline package/yast2-sap-ha.spec:20]: It downgrades the version...

**Review by lslezak — COMMENTED** [review]
It looks a bit strange, the version is downgraded

File: package/yast2-sap-ha.spec (hunk 3 above)
Comment (varkoly) [reply package/yast2-sap-ha.spec:20]: The reason is, that no newer 4.5 version was ever relesed. This is the first one.

**Pull request closed without merging**